Tests/StreamChatTests/Database/DTOs/ChannelReadDTO_Tests.swift

```diff
@@ -22,13 +22,41 @@ final class ChannelReadDTO_Tests: XCTestCase {
         super.tearDown()
     }
 
+    // MARK: - saveChannelRead
+
+    func test_saveChannelRead() throws {
```

Comment:
Is there any chance we can include a few words about what the expected outcome is?